cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -447,7 +452,35 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
         if (member == null && record is EnumDeclaration) {
             member = record.entries[reference.name.localName]
         }
-        return member ?: handleUnknownField(containingClass, reference)
+
+        if (member != null) {
+            return member
+        }
+
+        // This is a little bit of a workaround, but at least this makes sure we are not inferring a
+        // record, where a namespace already exist
+        val (scope, _) = scopeManager.extractScope(reference, null)
```

Comment:
This part is not really intuitive/understandable to me. Will we implement something for NameScope and else or does this have to remain as it is right now?

Reply:
Yes we will do that in the future